engine/src/main/java/org/terasology/logic/players/CameraClientSystem.java

```diff
@@ -65,15 +98,38 @@ public void resetCamera() {
     }
 
```

Comment:
Any time! Yep it is good to strike a balance between magic convenience and dark arcane wizardry :D